examples/if_ls_glob.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+
+TEST_DIR="test_dir"
```

Comment:
it is not clear what this variable for.

here and below - can you please add comments for every part that is not documented, so that newcomers could read the file without guessing what every line means, thanks

Reply:
thanks commend added!